experimental/adaptive-dialog/csharp_dotnetcore/todo-bot/Dialogs/AddToDoDialog/AddToDoDialog.cs

```diff
@@ -120,14 +134,20 @@ public AddToDoDialog()
             InitialDialogId = nameof(AdaptiveDialog);
         }
 
-        private static IRecognizer CreateRecognizer()
+        private static IRecognizer CreateRegeExRecognizer()
         {
             return new RegexRecognizer()
```

Comment:
>RegexRecognizer [](start = 23, length = 15)

BTW: RegexRecognizer can suport entity recognition now too.  Entities = new EntitySet() { new NumberTextRecognizer(), etc. }

Reply:
Yes. Added sample code here to highlight.

Reply:
Also updated doc.